front/app/components/FilterBoxes/TopicsFilter.tsx

```diff
@@ -140,48 +137,28 @@ const TopicsFilter = memo<Props>(
         >
           <Box>
             <Box className="e2e-topics-filters" aria-live="polite">
-              {topicsWithIdeas
-                .filter((topic) => !isError(topic))
```

Comment:
This is a remnant from the old days IIRC. Filtering is not needed anymore if our types are correct. (There's no guarantee they are, but they should be.)